aten/src/ATen/cudnn/Descriptors.h

```diff
@@ -278,6 +297,26 @@ struct TORCH_CUDA_CPP_API RNNDescriptor : public Descriptor<
             /*recProjSize=*/proj_size,
             /*outProjSize=*/0));
     }
+#else
+    auto md = mut_desc();
+    AT_CUDNN_CHECK(cudnnSetRNNDescriptor_v8(
+          md,
+          algo,
+          mode,
+          CUDNN_RNN_DOUBLE_BIAS,
+          bidirectional,
+          input_mode,
+          input_type,
+          datatype,
+          allow_tf32 ? CUDNN_DEFAULT_MATH : CUDNN_FMA_MATH,
+          input_size,
+          hidden_size,
+          proj_size ? proj_size : hidden_size,
+          num_layers,
+          dropout_desc_.desc(),
+          false ? CUDNN_RNN_PADDED_IO_DISABLED : CUDNN_RNN_PADDED_IO_ENABLED));
```

Comment:
dead code?

Reply:
good catch, this was from debugging earlier, reverted to be conditioned on `packed`